Associations/book.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '../Associations/rental'
+
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
```

Comment:
- [ ]  `rentals` should also include in attr_accessor.KIndly add it.

Reply:
done